src/hw-1/task1.ts

```diff
@@ -0,0 +1,24 @@
+import { stdin, stdout } from 'node:process';
+import { Stream } from 'stream';
+
+function getReversData(data: Buffer){
```

Comment:
it can accept encoding for not hardcoding this `utf-8`.

Reply:
Encoding is added on the scope of ReversStream.